plugins/scaffolder/src/components/fields/EntityNamePicker/validation.ts

```diff
@@ -22,6 +22,7 @@ export const entityNamePickerValidation = (
   validation: FieldValidation,
 ) => {
   if (!KubernetesValidatorFunctions.isValidObjectName(value)) {
+    // TODO I need access to i18n object to translate this. Hooks can't be called outside a Component Lifecycle
```

Comment:
You can use the `apiHolder` that's also passed into the context that you can do `apiHolder.get(translationApiRef)` to get a hold of the `TranslationApi`. You might have to subscribe to the observable to get a snapshot. I wonder if we should also make this a nice utility API to do this.

Reply:
Yes, I like the utility API approach, because we probably want to handle it in the same way for all the places and it's just boilerplate